### PR TITLE
Refactor the crash list to a Single File Component

### LINKED ISSUE #658: Vueify the crash list
Based on [the prototype](https://github.com/MozillaSecurity/FuzzManager/commit/521b225d802a372315c4c8a47ad927b5dcaa1a41) from @jschwartzentruber.

Improvements would be:
- convert to single file component
- use axios for request
- create sub component for crash display (the `<tr>` part)
- try using vue-router instead of manually updating the router
- add a shared `urls.js` file to build Django urls from clearly defined methods

Once approved by the team, replace the Django crash list by the vue.js version

### PR DESCRIPTION
**Pull request by EvaBardou**
Closes #658 

This PR is based on @jschwartzentruber prototype.

### CODE REVIEW

File: server/frontend/src/helpers.js [hunk 1]
```diff
@@ -32,3 +32,17 @@ export const errorParser = (error) => {
   if (typeof message === "string") return message;
   return message.toString();
 };
+
+export const E_SERVER_ERROR = "Error while communicating with the server.";
+
+export const formatClientTimestamp = (datetime) => {
+  return new Intl.DateTimeFormat("en-US", {
```
Comment (jschwartzentruber) [inline server/frontend/src/helpers.js:39]: Can you just change `"en-US"` -> `undefined`?

Testing this in console I get:

| locale | result |
|---|---|
| `"en-US"` | `5/12/2021, 09:13 EDT` |
| `"en-CA"` | `2021-05-12, 09:13 EDT` |
| `"fr"` | `12/05/2021 à 09:13 UTC−4` |
| `"de"` | `12.5.2021, 09:14 GMT-4` |
| `undefined` | `2021-05-12, 09:13 EDT` (same as `"en-CA"`, for me) |

**Review by jschwartzentruber — APPROVED** [review]
As discussed, I think you are planning to replace the Django `crashes` view with this implementation in the same MR, but this looks good to me so far.

**Comment by La0** [comment]
@jschwartzentruber If you could take another look at this Merge request... The previous implementation using Django views is now removed

**Review by jschwartzentruber — APPROVED** [review]
👍🏻

**Pull request merged**